packages/starknet-snap/src/types/snapApi.ts

```diff
@@ -146,3 +148,9 @@ export interface RpcV4GetTransactionReceiptResponse {
   execution_status?: string;
   finality_status?: string;
 }
+
+export interface SignTransactionParams extends BaseRequestParams, ExtractPrivateKeyRequestParams {
+  transactions: Call[];
```

Comment:
the data is pass from frontend, call[] type is the underlying type, please refer to SendTransactionRequestParams

Reply:
apply to the below following param as well

Reply:
TBC